view/common.css

```diff
@@ -13,7 +13,7 @@
 }
 
 body {
-  font: .9em/1.2em Source Sans Pro;
+  font: .9em/1.2em Source Sans Pro, Arial, Verdana, Helvetica, sans-serif;
```

Comment:
Please, keep only this line in your patch. The rest is not necessary.

Reply:
But i cleaned up the code format. I think it should be necessary